third_party/pyth/pyth_publisher.py

```diff
@@ -31,7 +31,7 @@ def do_GET(self):
         self.wfile.flush()
 
 # Test publisher state that gets served via the HTTP endpoint. Note: the schema of this dict is extended here and there
-HTTP_ENDPOINT_DATA = {"symbols": [], "mapping_address": None}
+HTTP_ENDPOINT_DATA = {"symbols": [], "mapping_address": None, "all_symbols_added": False}
```

Comment:
If I understand correctly, `all_symbols_added` makes the `check-attestations` dependency on `pyth` a special case. It is special, because an additional decision of the mock publisher  logic permits `check-attestations` to go green. 

I think that's reasonable, but can you note here what this is for?